### PR TITLE
Add support for tracking requests

### PR DESCRIPTION
**Pull request by lafka**
Enables tracking of  [Request Telemetry](https://docs.microsoft.com/en-us/azure/application-insights/application-insights-data-model-request-telemetry) (HTTP or otherwise).

### CODE REVIEW

**Comment by lafka** [comment]
I have some more changes related to adding tags (ie. `ai.user.id`) to all tracking api calls. Would you be interested in them? if so do you want them as separate PR or all combined?

**Comment by bottlenecked** [comment]
Hi @lafka thanks for the PR!

Could you please rename `resultCode` to `result_code` or `status_code` or something more Elixir-y anyway? I'll merge it right after.

**Comment by bottlenecked** [comment]
Also if it's all the same to you would you mind creating a separate pull request for the tags?
Thanks!

**Comment by lafka** [comment]
No problem i'll fix it -  mix of to many platforms these days make my coding conventions a bit all over the place:)

I'll fix the namings or this one and create a separate one for tags, will take a week or to due to some other priorities.

**Comment by DevExpDev** [comment]
Hi @lafka 
We looking to use the Request Telemetry. Can you please merge these changes

File: lib/data/payload.ex [hunk 1]
```diff
@@ -93,6 +93,22 @@ defmodule ExInsights.Data.Payload do
     |> create_payload("RemoteDependency")
   end
 
+  @doc """
+  Create request payload
+  """
+  def create_request_payload(name, url, source \\ nil, elapsed_time_ms, resultCode, success) do
```
Comment (DevExpDev) [inline lib/data/payload.ex]: Can we also support logging properties and measurements?

**Comment by bottlenecked** [comment]
Hi all, 

@DevExpDev I incorporated @lafka 's changes and published version 0.4.0. Closing this pull request

**Pull request closed without merging**